lib/facter/mongodb_version.rb

```diff
@@ -2,7 +2,7 @@
   setcode do
     if Facter::Core::Execution.which('mongo')
       mongodb_version = Facter::Core::Execution.execute('mongo --version 2>&1')
-      %r{^MongoDB shell version: ([\w\.]+)}.match(mongodb_version)[1]
+      %r{^MongoDB shell version:?\s+([\w\.]+)}.match(mongodb_version)[1]
```

Comment:
Adding v? will make the output look the same as the other versions (i.e. 3.4.0).